elements/pfe-avatar/src/pfe-avatar.js

```diff
@@ -155,9 +204,17 @@ class PfeAvatar extends PFElement {
     return hsl2rgb(...hsl);
   }
 
+  _updateWhenConnected() {
+    if (this.connected) {
+      this.update();
+    } else {
+      this.addEventListener(PfeAvatar.events.connected, () => this.update());
+    }
+  }
+
   update() {
     // if we have a src element, update the img, otherwise update the random pattern
-    if (this.hasAttribute("pfe-src")) {
+    if (this.src) {
```

Comment:
Is this documented?  That instead of `this.hasAttribute` we can now use the property instead?

Reply:
Sorry, I didn't see this message before.  It's documented that you _can_ reference properties this way, but not why you'd want to (it's a _lot_ less to type, and you get type conversions for free).  It would be a great thing to put in a tutorial.